src/main/java/ru/ayaz/Video.java

```diff
@@ -0,0 +1,38 @@
+package ru.ayaz;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+class Video {
+
+    private String videoName;
```

Comment:
Its not marked as an issue, rather it is a suggestion :)
No need to mark it as resolved as well.